.php_cs.dist

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__.'/src')
+    ->in(__DIR__.'/tests')
+;
+
+return PhpCsFixer\Config::create()
+    ->setUsingCache(true)
+    ->setRules([
+        '@Symfony' => true,
```

Comment:
I'd suggest using : 

```
return PhpCsFixer\Config::create()
    ->setRiskyAllowed(true)
    ->setRules([
        '@Symfony' => true,
        'array_syntax' => [
            'syntax' => 'short',
        ],
        'binary_operator_spaces' => [
            'align_double_arrow' => true,
            'align_equals' => true
        ],
        'braces' => [
            'allow_single_line_closure' => true,
        ],
    ])
    ->setFinder($finder)
;
```

RiskyRules can be removed (it's up to you), but `align_*` should not since it increase code readability a lot.

Reply:
I'd rather keep the default config as much as possible. I haven't seen many project enforcing these rules so I won't add them yet. We can can add them later if it becomes an issue.